notebooks/sprints/sprint19/530_rsclient_staging.ipynb

```diff
@@ -0,0 +1,520 @@
+    "    response_list.append(staging_client.run_staging(items, TEST_COLLECTION))"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "589c4f1e",
+   "metadata": {},
+   "outputs": [
+    {
+     "ename": "TypeError",
+     "evalue": "tuple indices must be integers or slices, not str",
+     "output_type": "error",
+     "traceback": [
+      "\u001b[0;31m---------------------------------------------------------------------------\u001b[0m",
+      "\u001b[0;31mTypeError\u001b[0m                                 Traceback (most recent call last)",
+      "Cell \u001b[0;32mIn[8], line 4\u001b[0m\n\u001b[1;32m      2\u001b[0m \u001b[38;5;28;01mfor\u001b[39;00m response \u001b[38;5;129;01min\u001b[39;00m response_list:\n\u001b[1;32m      3\u001b[0m     \u001b[38;5;28;01mwhile\u001b[39;00m timeout \u001b[38;5;241m>\u001b[39m \u001b[38;5;241m0\u001b[39m:\n\u001b[0;32m----> 4\u001b[0m         get_response \u001b[38;5;241m=\u001b[39m staging_client\u001b[38;5;241m.\u001b[39mget_job_status(\u001b[43mresponse\u001b[49m\u001b[43m[\u001b[49m\u001b[38;5;124;43m\"\u001b[39;49m\u001b[38;5;124;43mstatus\u001b[39;49m\u001b[38;5;124;43m\"\u001b[39;49m\u001b[43m]\u001b[49m[\u001b[38;5;124m\"\u001b[39m\u001b[38;5;124mstarted\u001b[39m\u001b[38;5;124m\"\u001b[39m])\n\u001b[1;32m      5\u001b[0m         \u001b[38;5;28;01mtry\u001b[39;00m:\n\u001b[1;32m      6\u001b[0m             resp \u001b[38;5;241m=\u001b[39m json\u001b[38;5;241m.\u001b[39mloads(get_response\u001b[38;5;241m.\u001b[39mcontent)\n",
+      "\u001b[0;31mTypeError\u001b[0m: tuple indices must be integers or slices, not str"
+     ]
+    }
+   ],
+   "source": [
+    "timeout = 120\n",
+    "for response in response_list:\n",
+    "    while timeout > 0:\n",
+    "        get_response = staging_client.get_job_status(response[1])\n",
+    "        try:\n",
+    "            resp = json.loads(get_response.content)\n",
+    "            pprint.PrettyPrinter(indent=4).pprint(resp)\n",
+    "            print(\"\\n\")\n",
+    "            if resp[\"status\"] == \"FINISHED\":\n",
+    "                print(\"Job COMPLETED\")\n",
+    "                break\n",
+    "\n",
+    "            if resp[\"status\"] == \"FAILED\":\n",
+    "                print(\"Job FAILED\")\n",
+    "                break\n",
+    "        except (json.JSONDecodeError,):\n",
+    "            continue\n",
+    "        time.sleep(2)\n",
+    "        timeout -= 2"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "d639dafa-db71-45a9-963c-4488ef5d08f1",
+   "metadata": {},
+   "source": [
+    "### Check the catalog for agrosu_my_test_collection. Four items should be present now"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "a77ea4cc",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Check the catalog for agrosu_my_test_collection\n",
+    "result = session.get(f\"{stac_client.href_catalog}/catalog/collections/{TEST_COLLECTION}/items\")\n",
```

Comment:
Instead of making an http request, you should use the rs-client-libraries wrapped function:
```python
stac_client.get_collection(COLLECTION).get_items()
```